propel/resource_data_pool.go

```diff
@@ -92,12 +92,6 @@ func resourceDataPool() *schema.Resource {
 					},
 				},
 			},
-			"tenant_id": {
```

Comment:
do you tested the update? Is there anything that we need to be aware of if a customer has that configured and then update to the version where is not supported?

Reply:
If a customer has `tenant` configured and tries to run terraform in the new version terraform will throw an error, I would say this is a breaking change so maybe we should let our customers know before they upgrade.

Not sure if terraform allows showing a warning instead in this case, IMO that would be the ideal.

Reply:
After removing the `tenant_id` and running apply in the new version it doesn't affect the existing data pools, those remain with the previously assigned tenant id so I think that part will work good for customers with an already configured data pool.